src/main/java/mods/eln/Eln.java

```diff
@@ -6645,6 +6647,12 @@ void recipeMiscItem() {
             Character.valueOf('g'), new ItemStack(Blocks.glass_pane),
             Character.valueOf('c'), new ItemStack(Items.dye, 1, 1));
 
+        addRecipe(findItemStack("Casing"),
```

Comment:
This is a bit expensive for what's a cosmetic item. Maybe make the recipe output 4 or 8 items?